addons/web/static/src/core/orm_service.js

```diff
@@ -335,6 +336,25 @@ export class ORM {
         validateObject("data", data);
         return this.call(model, "web_save", [ids, data], kwargs);
     }
+
+    /**
+     * @param {string} model
+     * @param {number[]} ids
+     * @param {any} data
+     * @param {any} [kwargs={}]
+     * @param {Object} [kwargs.specification]
+     * @param {Object} [kwargs.context]
+     * @returns {Promise<any[]>}
+     */
+    webSaveMulti(model, data, kwargs = {}) {
```

Comment:
I would have the same API as webSave:
```js
webSaveMulti(model, ids, data, kwargs = {}) {
```

Reply:
Sorry for insisting, but I don't understand why the logic is computed client side.

How does it work if we have a domain that translates to thousands of records?
Does the web client have to fetch the data of all the records, then compute the diff, then send everything back to the server? Seems counter intuitive to me.